Gemfile.lock

```diff
@@ -1,14 +1,12 @@
 GIT
```

Comment:
let's not commit the Gemfile.lock

Reply:
Ugh, do I make the commits again or can let it stay for this one (I'll remember not to commit it again)

Reply:
It's tricky right now because we're trying to keep the gemfile.lock on the most current master, instead of a version of fastlane, so I've noticed that when we commit the .lock file, sometimes I lose a critical change in fastlane/master. For right now, let's just back out that file 😞 It should get better in the future. If we commit this, as is, then fastlane.ci will break due to a missing commit on fastlane 🙃

Reply:
Alright, I'll restart the branch and commit again no problem 😄